test/testenv/appframework_utils.go

```diff
@@ -12,3 +16,22 @@ func GenerateAppSourceSpec(appSourceName string, appSourceLocation string, appSo
 		AppSourceDefaultSpec: appSourceDefaultSpec,
 	}
 }
+
+// GetAppStatus Get the app install status and enabled status
```

Comment:
This would only be applicable for locally installed apps and not clustered apps, correct?

Reply:
Yes.... Wokring on the part to include clustered apps as well

Reply:
Can we rename the method to reflect that ?